### PR TITLE
Admin Panel Scroll Issue & Core User's Organization Name Bug

### LINKED ISSUE #310: Adding/Updating permission does not show organization in user table
### **_Description:_**

When an Org Admin/Super Admin goes to the User Management section and either adds/updates permission to a user, the organization details for that user do not get populated in the organization column in the user table.

### **_Steps to recreate the issue:_**

1. Log in as Org Admin/Super Admin.
2. Navigate to User Management Section.
3. For new user, activate the user.
4. Add new permission/update existing permission for the user.

### PR DESCRIPTION
**Pull request by SiddharthBhansali13**
## Purpose
- Fix error form helper text color.
- Fix scroll issue on accordions in admin panel.
- Fix a bug which prevents a user's organization name to be populated after permission or active status update.

## Ticket number
Closes #310 & #311.

### CODE REVIEW

File: src/pages/UserManagement/Users/Users.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ const Users = ({ data, dispatch }) => {
                       label: 'Last activity',
                       prop: 'activity',
                       flex: '1',
-                      template: (row) => (
+                      template: () => (
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/Users/Users.js:226]: Why is the parameter row removed?

File: src/pages/UserManagement/Users/Users.js [hunk 2]
```diff
@@ -315,7 +315,7 @@ const Users = ({ data, dispatch }) => {
                               : null
                           }
                           >
-                            {organization.name}
+                            {organization.name ?? organization_name}
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/Users/Users.js:318]: I guess it should be `organization?.name || organization_name`

File: src/pages/UserManagement/Users/Users.js [hunk 3]
```diff
@@ -324,7 +324,7 @@ const Users = ({ data, dispatch }) => {
                       label: 'Last activity',
                       prop: 'activity',
                       flex: '1',
-                      template: (row) => (
+                      template: () => (
```
Comment (RadhikaPPatel) [inline src/pages/UserManagement/Users/Users.js:327]: Why is the parameter row removed?

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Please check the comments

File: src/pages/UserManagement/Users/Users.js (hunk 1 above)
Comment (SiddharthBhansali13) [reply src/pages/UserManagement/Users/Users.js:226]: The `row` parameter wasn't being used in the callback. Unused variable. Should I add it back?

File: src/pages/UserManagement/Users/Users.js (hunk 2 above)
Comment (SiddharthBhansali13) [reply src/pages/UserManagement/Users/Users.js:318]: Both will work the same. `??` will find the first non-null or non-undefined value and print it. Should I change?

File: src/pages/UserManagement/Users/Users.js (hunk 3 above)
Comment (SiddharthBhansali13) [reply src/pages/UserManagement/Users/Users.js:327]: Unused variable in the callback function.

**Review by RadhikaPPatel — APPROVED** [review]
Looks good

**Review by yasmin2496 — APPROVED** [review]

**Pull request merged**